packages/contracts-bedrock/src/L1/OptimismPortal2.sol

```diff
@@ -177,21 +181,21 @@ contract OptimismPortal2 is Initializable, ResourceMetering, ISemver {
     event RespectedGameTypeSet(GameType indexed newGameType, Timestamp indexed updatedAt);
 
     /// @notice Reverts when paused.
-    modifier whenNotPaused() {
+    function _whenNotPaused() private view {
```

Comment:
Is a requirement for reducing code size to make this function `private` instead of `internal`? If that's not the case, I suggest this change to enable the use of it from a contract that inherits from this one:
```suggestion
    function _whenNotPaused() internal view {
```

Reply:
Good catch!